.github/renovate.json

```
@@ -0,0 +1,29 @@
+{
+  "extends": [
+    "config:base",
+    ":gitSignOff",
+    "schedule:nonOfficeHours",
+    ":semanticCommitTypeAll(build)",
+    ":semanticCommitScope(deps)",
+    ":automergePatch"
+  ],
+  "addLabels": ["dependencies"],
+  "cloneSubmodules": true,
+  "prConcurrentLimit": 15,
+  "stabilityDays": 3,
+  "timezone": "Europe/Vienna",
+  "postUpdateOptions": [
+    "gomodTidy"
+  ],
+  "packageRules": [
+    {
+      "matchManagers": ["gomod"],
+      "addLabels": ["go"]
+    },
+    {
+      "matchManagers": ["github-actions"],
+      "addLabels": ["github_actions"]
+    }
+  ],
+  "commitBody": "[skip ci]"
```

Please use [commitMessageSuffix](https://github.com/mowies/kubescape-color-test/blob/main/.github/workflows/test.yml) instead here, otherwise GitHub Actions will not pick this up correctly I think :)
```suggestion
  "commitMessageSuffix": " [skip ci]"
```